recipe/meta.yaml

```diff
@@ -36,12 +41,12 @@ test:
     - jupyter nbextension list 1>nbextensions 2>&1
     - jupyter labextension list
     - jupyter labextension list 1>labextensions 2>&1
-    - grep -iE "jupyter_bokeh/extension.*enabled" nbextensions
-    - grep -iE "@bokeh/jupyter_bokeh.*OK.*jupyter_bokeh" labextensions
+    - rg -ie "jupyter_bokeh/extension.*enabled" nbextensions
+    - rg -ie "@bokeh/jupyter_bokeh.*OK.*jupyter_bokeh" labextensions
   requires:
     - pip
     - jupyterlab
-    - m2-grep  # [win]
+    - ripgrep
 
 about:
   home: https://pypi.org/project/jupyter-bokeh
```

Comment:
It might be good to add the following to the about section:

description
license_family
doc_url

Reply:
done